deploy/helm/sumologic/values.yaml

```diff
@@ -1949,6 +1949,47 @@ kube-prometheus-stack:
               regex: (?:kafka_(broker_.*|controller_.*|java_lang_.*|partition_.*|purgatory_.*|network_.*|replica_.*|request_.*|topic_.*|topics_.*|zookeeper_.*))
               sourceLabels: [__name__]
 
+        ## MySQL Telegraf Metrics
+        ## List of Metrics are on following github page:
+        ## https://github.com/influxdata/telegraf/tree/master/plugins/inputs/mysql#metrics
```

Comment:
`master` will move with time and we might end up with an incorrect link. Can we pin it to a concrete tag? e.g. 
https://github.com/influxdata/telegraf/tree/v1.18.1/plugins/inputs/mysql#metrics ?

Reply:
Done